### ISSUE TITLE
Image ignores soft text wrapping

### ISSUE THREAD

**Issue by Vlusky** [3rd/image.nvim#116]
I use soft wrapping for my paragraphs but it seems like it had trouble locating where it should be.
(using Kitty back end)

https://github.com/3rd/image.nvim/assets/90129039/ec458928-40b7-4c64-880b-ad8a5a306b67

**Comment by 3rd** [comment]
Yeah, this is not implemented, there's no performant API in nvim to measure the rendered height of a buffer section :|

**Comment by 3rd** [comment]
we have this now
